### PR TITLE
don't mangle config.GinkgoConfig.FocusString

### PR DESCRIPTION
**Pull request by jim-minter**
The logic attempting to modify config.GinkgoConfig.FocusString looks wrong to me.  It looks like it's attempting to use alternation, however after appending "|" it overwrites the string.  In any case, I think boolean AND is wanted, not alternation, and this is not implementable using Go regexes.

### CODE REVIEW

**Comment by jim-minter** [comment]
@smarterclayton ptal
@miminar does this also achieve what you were setting out to do in #16806?

**Comment by jim-minter** [comment]
or @bparees ptal

File: test/extended/util/test.go [hunk 1]
```diff
@@ -128,29 +116,40 @@ func ExecuteTest(t *testing.T, suite string) {
...
 			}
 		}
 
-		if !excludedTestsFilter.MatchString(name) {
-			include := conformanceTestsFilter.MatchString(name)
-			switch {
-			case !include:
-				// do nothing
-			case isSerial:
+		if !isExcluded && isConformance {
```
Comment (smarterclayton) [inline test/extended/util/test.go]: Please don't refactor this chunk of code, it's orthogonal to your change.

File: test/extended/util/test.go [hunk 2]
```diff
@@ -128,29 +116,40 @@ func ExecuteTest(t *testing.T, suite string) {
...
 		}
+
 		if strings.Contains(node.CodeLocation().FileName, "/origin/test/") && !strings.Contains(node.Text(), "[Suite:openshift") {
 			node.SetText(node.Text() + " [Suite:openshift]")
 		}
 		if strings.Contains(node.CodeLocation().FileName, "/kubernetes/test/e2e/") {
 			node.SetText(node.Text() + " [Suite:k8s]")
 		}
+
+		switch syntheticSuite {
```
Comment (smarterclayton) [inline test/extended/util/test.go]: Please do not do this.

**Comment by smarterclayton** [comment]
If it's not implementable via regexes I don't want to do it via hacking the pending status of flags either.

I'm considering removing the serial suite altogether from the job so that this is not an issue. Do builds have any serial components?

**Comment by jim-minter** [comment]
The only one in builds is test/extended/builds/run_policy.go

```
// this test is very latency sensitive so run it by itself (serially).
var _ = g.Describe("[Feature:Builds][Slow][Serial] using build configuration runPolicy", func() {
```

#16262

**Pull request closed without merging**